<?xml version="1.0"?>
<launch>
    <!--Launch Zed Wrapper-->
	<include file="$(find zed_wrapper)/launch/zed0.launch"/>

    <!--republish-->
    <node name="republish0" type="republish" pkg="image_transport" output="screen" 
          args="compressed in:=/zed0/left/image_rect_color raw out:=/zed0/left/image_rect_color/republish" />

    <!--Coloring PointCloud-->
	<param name="target_frame"  type="string" value="/zed0/zed_left_camera" />
	<param name="source_frame"  type="string" value="/centerlaser" />
	<param name="debug" 		type="bool"   value="false" />
	<node pkg="sensor_fusion" type="coloringcloud" name="coloring_zed0">
		<remap from="/cloud" 		 to="/sq_lidar/points/center" />
		<remap from="/camera_info" 	 to="/zed0/left/camera_info" />
		<remap from="/image" 		 to="/zed0/left/image_rect_color/republish" />
		<remap from="/cloud/colored" to="/zed0/colored" />
	</node>
</launch>
